From 0edff1329b55d2a32ff0639352fc7e517b4e067e Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sat, 22 Jun 2024 15:44:23 -0700
Subject: [PATCH] Docs/maintainer/maintainer-entry-profile: add DAMON
 maintainer profile

DAMON has its maintainer entry profile document, but it's not listed on
the existing profiles section of maintainer-entry-profile.rst.  Add it.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/maintainer/maintainer-entry-profile.rst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/maintainer/maintainer-entry-profile.rst b/Documentation/maintainer/maintainer-entry-profile.rst
index b49fb6dc4d0c..cda5d691e967 100644
--- a/Documentation/maintainer/maintainer-entry-profile.rst
+++ b/Documentation/maintainer/maintainer-entry-profile.rst
@@ -109,3 +109,4 @@ to do something different in the near future.
    ../driver-api/vfio-pci-device-specific-driver-acceptance
    ../nvme/feature-and-quirk-policy
    ../filesystems/xfs/xfs-maintainer-entry-profile
+   ../mm/damon/maintainer-profile
-- 
2.39.2

